### ISSUE TITLE
Can we allow text colour to be changed as well? 

### ISSUE THREAD

**Issue by ChadTaljaardt** [briandowns/spinner#86]
I don't see in the documentation any way to change the colour of the text for the spinner. Is there a way this can be done?

**Comment by briandowns** [comment]
Hi @ChadTaljaardt, it doesn't appear to be a way once the spinner has started. I can take a look at adding feature unless you wanted to submit a PR. :)

**Comment by ChadTaljaardt** [comment]
Do you know of a way before the spinner has started?  I am looking to have either the prefix text or suffix text to be coloured. 

I can look into making a PR on the weekend if i am free.

**Comment by theckman** [comment]
@ChadTaljaardt because of this exact issue I ended up rolling my own spinner starting at the end of the year, [yacspin](https://github.com/theckman/yacspin).

I had looked at contributing this functionality to this package, but with the way the spinner was structured it felt like I'd need to make some major changes to do it in a thread-safe way. That made me yak shave my own CLi spinner, which probably ended up being more work. 😅
